samples/cpp/classification_sample_async/main.cpp

```diff
@@ -62,6 +55,20 @@ bool parse_and_check_command_line(int argc, char* argv[]) {
     return true;
 }
 
+ov::Layout get_layout_from_shape(const ov::Shape& shape) {
```

Comment:
If that way of layout extraction is a work around, what is the correct way to do that?

There must be layout extraction anyway. Either to set appropriate layout or validate that the model layout is what the sample expects, otherwise the error message is incorrect.